### PR TITLE
added sensitive property

### PR DESCRIPTION
**Pull request by SamMarkowitz**
fixes CloudSlang/cloud-slang#467

### CODE REVIEW

**Comment by SamMarkowitz** [comment]
@mihaitusa, if you have a minute, please take a look to see that I have the right idea.

**Comment by SamMarkowitz** [comment]
Build errors as in #653. Waiting on their resolution.

File: content/io/cloudslang/base/http/http_client_action.sl [hunk 1]
```diff
@@ -115,43 +115,55 @@ operation:
         private: true
     - username:
         required: false
+        sensitive: true
```
Comment (Bonczidai) [inline content/io/cloudslang/base/http/http_client_action.sl]: together with @mihaitusa we think only password fields like `password`, `proxy_password`, `trustPassword` should be sensitive -- is there a specific reason making other values sensitive? (is this a convention we are going to use?)

Comment (SamMarkowitz) [inline content/io/cloudslang/base/http/http_client_action.sl]: Got it. Will remove the others.

**Comment by SamMarkowitz** [comment]
@mihaitusa I have one build error still. I'm not sure what's causing it. Can you take a look?

**Comment by SamMarkowitz** [comment]
Strange, I was sure I did that already, but it turns out I didn't. Let's hope it works.

**Pull request merged**